incubator/minecraft/templates/deployment.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+      - name: {{ template "fullname" . }}
+        image: "{{ .Values.Image }}:{{ .Values.ImageTag }}"
+        imagePullPolicy: Always
+        resources:
+          requests:
+            cpu: "{{.Values.Cpu}}"
+            memory: "{{.Values.Memory}}"
+          limits:
+            cpu: "{{.Values.Cpu}}"
```

Comment:
@gtaylor rather than having individual values for each you can make the entire dict a value as shown here:
https://github.com/kubernetes/charts/blob/master/stable/wordpress/values.yaml#L92

I prefer that method. Your defaults can stay the same as you have them now.